gini-internal-payment-sdk/library/src/main/java/net/gini/android/internal/payment/util/extensions/Fragment.kt

```diff
@@ -0,0 +1,18 @@
+package net.gini.android.internal.payment.util.extensions
+
+import android.view.LayoutInflater
+import androidx.fragment.app.Fragment
+import net.gini.android.internal.payment.GiniInternalPaymentModule
+import java.util.Locale
+
+internal fun Fragment.getLayoutInflaterWithGiniHealthThemeAndLocale(inflater: LayoutInflater, locale: Locale? = null): LayoutInflater {
```

Comment:
[Request] Please update this to `getLayoutInflaterWithGiniPaymentThemeAndLocale`